test/polar/Pie.spec.tsx

```diff
@@ -272,17 +271,7 @@ describe('<Pie />', () => {
       outerRadius: 200,
       paddingAngle: 0,
       payload: {
-        cx: 250,
```

Comment:
not sure how we want all of this to look. this gets rid of `payload.payload` which has never really made sense to me... You have all of the information, its just 1 level above.

I probably need to figure this out more

Reply:
I'm all in for deleting it. Shouldn't have been there in the first place